.github/workflows/ci.yml

```diff
@@ -597,6 +597,18 @@ jobs:
           repository: sass/sass-site
           token: ${{ secrets.SASS_SITE_TOKEN }}
 
+      - name: Get version
+        id: version
+        run: echo "version=${GITHUB_REF##*/}" | tee --append "$GITHUB_OUTPUT"
+
+      - name: Update Dart Sass version
+        run: |
+          cat package.json |
+            jq --arg version ^${{ steps.version.outputs.version }} '
+              .devDependencies.sass |= $version
+            ' > package.json.tmp
+          mv package.json.tmp package.json
```

Comment:
As there is a committed lock file, this won't be enough

Reply:
We can run `npm update sass` after that, which should update only `sass` in the package-lock.json.

However, this job should depends on `deploy_npm` so that it's updated only after npm package has been published.